docs/Gemfile.lock

```diff
@@ -48,7 +48,7 @@ GEM
       rb-inotify (>= 0.9)
     mercenary (0.3.4)
     mini_portile (0.6.0)
-    nokogiri (1.6.3.1)
+    nokogiri (1.6.6.2)
```

Comment:
Unfortunately the `.lock` file is built from the `Gemfile` so this might cause issues for others. I *hate* nokogiri so, sorry for the troubles.